release_util/management/commands/__init__.py

```diff
@@ -342,22 +378,26 @@ def __apply(self, migration=None, run_all=False):
         start = self._timer()
         try:
             call_command("migrate", **migrate_kwargs)
-        except (CommandError, DatabaseError):
-            trace = repr(traceback.format_exception(*sys.exc_info()))
-
-        end = self._timer()
-        successes, failure = self._parse_migrate_output(out.getvalue())
-
-        self._migration_state.append({
-            'migration': 'all' if run_all else (migration[0], migration[1]),
-            'duration': end - start,
-            'output': _remove_escape_characters(out.getvalue()),
-            'success': successes,        # [(app, migration), ...]
-            'failure': failure,          # (app, migration)
-            'traceback': trace,
-        })
-        if trace is not None:
+        except:
```

Comment:
Unfortunately in this case we actually want a bare except because we're just passing the traceback through. We're not doing anything based on the type of exception and we need to swallow all exceptions.